http-server-upload.js

```diff
@@ -217,8 +218,8 @@ server.on('request', (req, res) => {
   <title>http-server-upload</title>
 </head>
 <body>
-<form action="upload" method="post" enctype="multipart/form-data">
-  Files: <input type="file" name="uploads" multiple="multiple"><br />
+<form action="upload" method="post" enctype="multipart/form-data" onsubmit="let totalSize=0; for (let file of document.getElementById('${formId}').files){totalSize+=file.size} if(totalSize){return (totalSize<${maxFileSize})?true:(alert(\`Cannot upload. Input files \${(totalSize/1024/1024).toFixed(2)} MB exceed ${maxFileSize/1024/1024} MB limit.\`),false) }else{alert('No file selected.');return false}">
```

Comment:
What about uploading files with a size of 0 bytes?
In some cases 0 bytes files may be used for signaling.

Reply:
:facepalm: missed that